### PR TITLE
Typo Correction. Also more Consts added. Header cleaning.

### PR DESCRIPTION
**Pull request by donnyh13**

### CODE REVIEW

File: LibreOfficeWriter_Font.au3 [hunk 1]
```diff
@@ -34,8 +34,7 @@
 ; Name ..........: _LOWriter_FontExists
 ; Description ...: Tests whether a Document has a specific font available by name.
 ; Syntax ........: _LOWriter_FontExists(Byref $oDoc, $sFontName)
-; Parameters ....: $oDoc           - [in/out] an object. A Document object returned by previous DocOpen, DocConnect, or
-;				   +					DocCreate function.
+; Parameters ....: $oDoc           - [in/out] an object. A Document object returned by previous DocOpen, DocConnect, or DocCreate function.
```
Comment (mlipok) [inline LibreOfficeWriter_Font.au3:37]: full function names
`_LOWriter_DocOpen, _LOWriter_DocConnect, or _LOWriter_DocCreate`

Please check also other places.

File: LibreOfficeWriter_Char.au3 [hunk 2]
```diff
@@ -12,7 +12,7 @@
 ; AutoIt Version : v3.3.16.1
 ; UDF Version    : 0.0.0.3
 ; Description ...: Provides basic functionality through Autoit for interacting with Libre Office Writer.
-; Author(s) .....: donnyh13
+; Author(s) .....: donnyh13 mLipok
```
Comment (mlipok) [inline LibreOfficeWriter_Char.au3]: comma
`; Author(s) .....: donnyh13, mLipok`

Please check also other places.

File: LibreOfficeWriter_Char.au3 [hunk 3]
```diff
@@ -841,19 +740,15 @@ EndFunc   ;==>_LOWriter_CharStyleGetObj
 ;				   @Error 1 @Extended 7 Return 0 = $sParentStyle Doesn't exist in this Document.
 ;				   @Error 1 @Extended 8 Return 0 = $bHidden not a Boolean.
 ;				   --Property Setting Errors--
-;				   @Error 4 @Extended ? Return 0 = Some settings were not successfully set. Use BitAND to test @Extended for
-;				   +								the following values: 1, 2, 4
+;				   @Error 4 @Extended ? Return 0 = Some settings were not successfully set. Use BitAND to test @Extended for the following values: 1, 2, 4
```
Comment (mlipok) [inline LibreOfficeWriter_Char.au3]: Please use
```
the following values:
```
instead:
```
the following values: 1, 2, 4
```

there is no need to enumarate values when you have them listed below.

Please check also other places.

File: LibreOfficeWriter_Char.au3 [hunk 4]
```diff
@@ -1133,11 +1006,8 @@ EndFunc   ;==>_LOWriter_CharStyleRotateScale
 ; Name ..........: _LOWriter_CharStyleSet
 ; Description ...: Set a Character style for a section of text by Cursor or paragraph Object.
 ; Syntax ........: _LOWriter_CharStyleSet(Byref $oDoc, Byref $oObj, $sCharStyle)
-; Parameters ....: $oDoc           - [in/out] an object. A Document object returned by previous DocOpen, DocConnect, or
-;				   +					DocCreate function.
-;                  $oObj                - [in/out] an object. A Cursor Object returned from any Cursor Object creation
-;				   +						or retrieval functions, Or A Paragraph Object returned from
-;				   +						_LOWriter_ParObjCreateList function.
+; Parameters ....: $oDoc           - [in/out] an object. A Document object returned by previous DocOpen, DocConnect, or DocCreate function.
```
Comment (mlipok) [inline LibreOfficeWriter_Char.au3]: Please use the same column location
image

Please check also other places.

File: LibreOfficeWriter_Char.au3 (hunk 2 above)
Comment (donnyh13) [reply LibreOfficeWriter_Char.au3]: Done

**Pull request merged**